demos/background_subtraction_demo/cpp_gapi/main.cpp

```diff
@@ -0,0 +1,184 @@
+            // it's needed only to blur user background, otherwise second input will be used.
+            cv::optional<cv::GMat> target_bgr;
+
+            cv::GMat bgr_resized;
+            bool is_blur = FLAGS_blur_bgr != 0;
+            if (is_blur && FLAGS_target_bgr.empty()) {
+                bgr_resized = in;
+            } else {
+                target_bgr = cv::util::make_optional<cv::GMat>(cv::GMat());
+                bgr_resized = custom::GResize::on(target_bgr.value(), in);
+            }
+
+            cv::GMat blob = cv::gapi::infer<custom::BackgroundMattingModel>(in);
+            auto alpha    = custom::GResize::on(custom::GTensorToImg::on(blob), in);
+            auto alpha3ch = cv::gapi::merge3(alpha, alpha, alpha);
+
+            auto bgr = FLAGS_blur_bgr
+                ? cv::gapi::blur(bgr_resized, cv::Size(FLAGS_blur_bgr, FLAGS_blur_bgr))
+                : bgr_resized;
+
+            auto in_fp  = cv::gapi::convertTo(in, CV_32F);
+            auto bgr_fp = cv::gapi::convertTo(bgr, CV_32F);
+
+            cv::GScalar one(cv::Scalar::all(1.));
+            auto out = cv::gapi::mul(alpha3ch, in_fp) + cv::gapi::mul((one - alpha3ch), bgr_fp);
+            auto result = cv::gapi::convertTo(out, CV_8U);
+
+            auto graph_inputs = cv::GIn(in);
+            if (target_bgr.has_value()) {
+                graph_inputs += cv::GIn(target_bgr.value());
+            }
+
+            return cv::GComputation(std::move(graph_inputs), cv::GOut(result));
+        });
+
+        /** Configure network **/
+        auto config = ConfigFactory::getUserConfig(FLAGS_d, "", "", FLAGS_nireq,
+                                                   FLAGS_nstreams, FLAGS_nthreads);
+        const auto net = cv::gapi::ie::Params<custom::BackgroundMattingModel> {
+            FLAGS_m,                           // path to topology IR
+            fileNameNoExt(FLAGS_m) + ".bin",   // path to weights
+            FLAGS_d                            // device specifier
+        }.cfgNumRequests(config.maxAsyncRequests)
+         .pluginConfig(config.execNetworkConfig);
+        slog::info << "The background matting model " << FLAGS_m << " is loaded to " << FLAGS_d << " device." << slog::endl;
+
+        /** Get information about frame **/
+        std::shared_ptr<ImagesCapture> cap = openImagesCapture(FLAGS_i, FLAGS_loop, 0,
+            std::numeric_limits<size_t>::max(), stringToSize(FLAGS_res));
+        const auto tmp = cap->read();
+        if (!tmp.data) {
+            throw std::runtime_error("Couldn't grab first frame");
+        }
+        cv::Size frame_size = cv::Size{tmp.cols, tmp.rows};
+        cap = openImagesCapture(FLAGS_i, FLAGS_loop, 0,
+            std::numeric_limits<size_t>::max(), stringToSize(FLAGS_res));
+
+        auto pipeline = comp.compileStreaming(cv::compile_args(custom::kernels(),
+                                                               cv::gapi::networks(net)));
+
+        /** Output containers for results **/
```

Comment:
```suggestion
        /** Output container for result **/
```
Is there should be a singular form, isn't it?

Reply:
Yep, done